setup.py

```diff
@@ -251,16 +251,23 @@ def bazel_build(self, ext):
       bazel_argv.append('--config=tpu')
 
     # Remote cache authentication.
-    if _check_env_flag('BAZEL_REMOTE_CACHE'):
-      bazel_argv.append('--config=remote_cache')
-
-    if GCLOUD_KEY_FILE:
-      bazel_argv.append('--google_credentials=%s' % GCLOUD_KEY_FILE)
-      if not _check_env_flag('BAZEL_REMOTE_CACHE'):
+    gclout_key_file_size = os.path.getsize(GCLOUD_KEY_FILE)
+    # Temporary workaround to allow PRs from forked repo to run CI. See details at (#5259).
+    # TODO: Remove the check once self-hosted GHA workers are avaialble to CPU/GPU CI.
+    print("check GCLOUD_KEY_FILE: {}".format(GCLOUD_KEY_FILE))
+    print("check if file exists: {}".format(os.path.exists(GCLOUD_KEY_FILE)))
+    print("check size: {}".format(os.path.getsize(GCLOUD_KEY_FILE)))
+    if gclout_key_file_size > 1:
```

Comment:
Won't this prevent us from using the remove cache if we don't have a `GCLOUD_KEY_FILE` set? I've been running the remote cache with my GCE machine's default credentials, not a key file.

Reply:
Yes, there thw adc is implicit. The criterion could be iff the key file.env var is set to an existing file that has size 1 or less, turn it off.